meta/Makefile

```diff
@@ -95,8 +95,15 @@ toolsversions:
 
 CONSTHEADERS = saimetadatatypes.h saimetadatalogger.h saimetadatautils.h saiserialize.h
 
+DOXYGEN_VERSION_CHECK = $(shell echo -e "$$(doxygen -v)\n1.9.0" | sort -V | head -n1)
+ifeq (${DOXYGEN_VERSION_CHECK},1.9.0)
```

Comment:
this is only checking one version, and will fall back if doxygen changes to let say 1.9.1 and error will still be shown, and actually new lines "^^" are introduced from doxygen 1.8.16

Reply:
i yesterday thought about this how make this more generic compare, but i didn't found easy and compact way to compare version strings, the closes way i found was perl version:
```
#!/usr/bin/perl
use version;
print version->parse(1.8.15) < version->parse(1.9.0);
```

Reply:
this could be actually moved to separate perl file, and that file could be used to output result for Makefile ifeq check

Reply:
Hi Kamil `sort -V` compares versions this should be extendable.

Reply:
> Hi Kamil `sort -V` compares versions this should be extendable.

yes, it compares, but you still compare EQ to 1.9.0, so this will fail for 1.9.1, so operator "<" should be used, thats why i proposed perl version compare which does exactly that

Reply:
It is using `sort` against version `1.9.0` and the version of Doxygen. This command will print the lower version first and then the higher version. The code checks that the lower version is `1.9.0` which means that the version of Doxygen is `1.9.0` or higher which is what we are intending to test. 

I have tested this change already in 1.9.1 and it was successful.

Reply:
this still does not solve issue for version v1.8.16 which should use Doxygen file, and v1.8.15 should use Doxygen.compat, it will only be working from version 1.9.0 and above, you would need to use 1.8.16 string instead 1.9.0